README.md

```diff
@@ -868,6 +869,56 @@ Merobox provides automatic Docker image management to ensure your workflows alwa
 - `CALIMERO_AUTH_FRONTEND_FETCH`: Set to `0` to use cached auth frontend (default is `1` for fresh fetch)
 - `CALIMERO_WEBUI_FETCH`: Set to `0` to use cached WebUI frontend (default is `1` for fresh fetch)
 
+### Custom Config Path
+
+Merobox allows you to specify custom `config.toml` files for workflow nodes, enabling you to reuse existing node configurations without relying on Docker image initialization.
+
+#### **Usage in Workflows**
+
+You can specify a custom config path at two levels:
+
+**Shared config for all nodes:**
+
+```yaml
+nodes:
+  config_path: ./custom-config.toml  # All nodes use this config
+  calimero-node-1:
+    port: 2428
+    rpc_port: 2528
+  calimero-node-2:
+    port: 2429
+    rpc_port: 2529
+```
+
+**Per-node config override:**
+
+```yaml
+nodes:
+  config_path: ./default-config.toml  # Default for all nodes
+  calimero-node-1:
+    port: 2428
+    rpc_port: 2528
+    # Uses default config
+  calimero-node-2:
+    port: 2429
+    rpc_port: 2529
+    config_path: ./special-config.toml  # Override for this node
+```
+
+#### **Features**
+
+- **Shared or Per-Node**: Specify a default config for all nodes or override per-node
+- **Skip Initialization**: When custom config is provided, node initialization is skipped
+- **Docker & Binary Mode**: Works with both Docker containers and native binary (`--no-docker`) mode
+- **Path Resolution**: Supports both absolute and relative paths
+- **Validation**: Validates config file existence before attempting to use it
+
+#### **Example**
+
+See `workflow-examples/workflow-custom-config-example.yml` and `workflow-examples/custom-config.toml` for a complete example.
+
+**Note:** Custom config path is not supported with `count` mode (bulk node creation).
```

Comment:
I think, if this happens, we need to explicitly provide an error, otherwise we'd have a silent ignoring, like we had with the node count and node names before. So here the same - we might expect the config to apply to the bulk nodes, but it is ignored.

Let's explicitly check if `config_path` is present when `count > 0` and either throw an error on yaml file validation, or rather support it.